Configs/.zshrc

```diff
@@ -63,13 +63,15 @@ alias  l='eza -lh  --icons=auto' # long list
 alias ls='eza -1   --icons=auto' # short list
 alias ll='eza -lha --icons=auto --sort=name --group-directories-first' # long list all
 alias ld='eza -lhD --icons=auto' # long list dirs
+alias lt='eza --icons=auto --tree' # list folder as tree
 alias un='$aurhelper -Rns' # uninstall package
 alias up='$aurhelper -Syu' # update system/package/aur
 alias pl='$aurhelper -Qs' # list installed package
 alias pa='$aurhelper -Ss' # list availabe package
 alias pc='$aurhelper -Sc' # remove unused cache
 alias po='$aurhelper -Qtdq | $aurhelper -Rns -' # remove unused packages, also try > $aurhelper -Qqd | $aurhelper -Rsu --print -
 alias vc='code' # gui code editor
+alias neofetch='fastfetch' # alias neofetch as fastfetch for backword compatibility for users
```

Comment:
wait... why? I think this would be misleading or confusing...if `neofetch` is not installed then it should just fail

![image](https://github.com/prasanthrangan/hyprdots/assets/106020512/b2fccd4d-b54c-400d-be37-0bcc2a3ad907)

Reply:
SUre will remove it. Thought people who were using neofetch on hyperdots would be having trouble migrating to fastfetch